src/Chapter1.hs

```diff
@@ -632,7 +650,14 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = newLastDigit n + secondLastDigit n
+  where
+    newLastDigit :: Int -> Int
+    newLastDigit x = mod (abs x) 10
+
+    secondLastDigit :: Int -> Int
+    secondLastDigit x = lastDigit (div (abs x) 10)
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod m 10
div m 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(x, y) = divMod m 10
```

You can see how we could pattern match on the pair 🙂

Reply:
That's cool, just switched the solution to use `divMod`! Pattern matching is an awesome feature I miss in many languages 😞